test/configs/digests.config

```diff
@@ -48,3 +48,15 @@ digests = [
     }
 ]
 
+[[FileDigestGroup]]
+packages = ["otherpkg", "testpkg2"]
+type = "somerestriction"
+note = "tests a package that has matching digests, using multiple package names"
+bug = "bsc#1234"
+digests = [
+    {
+        path = "/resetricted/1/afile",
```

Comment:
It's because of this typo: `resetricted`, should be `restricted`.

Reply:
Thanks for the hint. I must have been making some major mistake when running the tests. It should be correct now.